### ISSUE TITLE
Troubles with the call to /api/v1/mutes

### ISSUE THREAD

**Issue by stom79** [tootsuite/mastodon#5588]
Hello,

The call to `/api/v1/mutes` is returning the same value for `since_id` and `max_id` in the  link header, which creates strange behaviors in some instances (infinite scrolling with same values).

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by stom79** [comment]
I have no Ruby programing knowledge but I wonder if [this piece of code](https://github.com/tootsuite/mastodon/blob/8b2cad56374b2dbb6e7a445e7917810935c45536/app/controllers/api/v1/mutes_controller.rb#L49-L55) is correct for getting the `max_id` and `since_id`: 

 ```ruby
 def pagination_max_id
    @accounts.last.muted_by_ids.last
  end

  def pagination_since_id
    @accounts.first.muted_by_ids.first
  end
```

Should not it be something like: ?

```ruby
 def pagination_max_id
    @accounts.first.muted_by_ids.last
  end

  def pagination_since_id
    @accounts.first.muted_by_ids.first
  end
```

**Comment by JMendyk** [comment]
@stom79 
Yesterday similiar issue #8463 was reported and I have fixed it in #8472. Indeed, as you suggested, `pagination_max_id` and `pagination_since_id` didn't work correctly. Can this issue be closed now?
